src/AwsLambda/AwsLambdaOpenTracer/Util/ConcreteFileManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.IO;
+
+namespace NewRelic.OpenTracing.AmazonLambda
+{
+    interface IFileManager
+    {
+        bool Exists(string path);
+        void WriteAllText(string path, string contents);
+    }
+
+    public class ConcreteFileManager : IFileManager
```

Comment:
`ConcreteFileManager` was just an example class name in my pseudocode. I'm thinking something without the word `Concrete` in it,  like `FileSystemManager` ?